README.md

```diff
@@ -62,6 +62,10 @@ To install OADP operator and the essential Velero components follow the steps gi
   ```
   oc project oadp-operator
   ```
+- Create secret for the cloud provider credentials to be used
+  ```
+  oc create secret generic <SECRET_NAME> --namespace oadp-operator --from-file cloud=<CREDENTIALS_FILE_PATH>
```

Comment:
Would be nice if we included a template link: https://github.com/konveyor/velero-examples/blob/master/velero-install/aws-credentials. Just say "`CREDENTIALS_FILE_PATH` should be in proper AWS creds format, for reference see: <file>"

Reply:
Done. Updated readme.